### ISSUE TITLE
Accessibility: Improve contrast of homepage text.

### ISSUE THREAD

**Issue by NedReid** [NedReid/ButlerJCRWebsite#77]
**AC1:**
**WHEN:** I run lighthouse or pagespeed insights on the homepage
**THEN:** There should be no issues with contrast under the "accessibility" section

You may think this is a ticket about accessibility. Psych! 😝  This is actually a ticket about **S E O**.

Right now we get a pitiful **96** for accessibility in [pagespeed insights](https://pagespeed.web.dev/analysis/https-butlerjcr-com/oypgqcvu0r?form_factor=mobile), and this is almost solely due to the bad contrast on the test "Welcome to Butler JCR!"

<img width="1540" alt="image" src="https://github.com/NedReid/ButlerJCRWebsite/assets/56114384/b531b5c7-58f7-48f0-bdca-6a53074f32c8">

This isn't good enough, and could be the difference between being at the top and bottom of the search results! (though very unlikely!)

**Technical Elaboration**
- I like the way it looks right now and tbh I think the contrast already seems fine.
- Maybe having a slight outline on the text would improve contrast.
- Try play around and see if anything else can help improve it!
